internal/types/types.go

```diff
@@ -48,6 +49,7 @@ type ErrIgnore struct {
 	Error KnownError `toml:"error"`
 	Files []string   `toml:"files"`
 	Dirs  []string   `toml:"dirs"`
+	Tags  []string   `toml:"tags"`
```

Comment:
Looks like this adds an ability to ignore a particular error for a particular tag. If yes, does it work for other checks?

Reply:
No - I believe it's used exclusively for checking the distribution.

Reply:
Maybe add a comment about that, as it is counter-intuitive.